modules/importing-repositories/procedure-importing-multiple-repositories-from-github.adoc

```diff
@@ -0,0 +1,36 @@
+= Importing multiple repositories
+
+The Bulk Import feature allows you to view all the GitHub repositories and organizations accessible from the configured GitHub Integrations.
+From there, you can pick any number of repositories and onboard them.
+
+.Prerequisites
+* You have xref:enabling-ang-giving-access-to-the-bulk-import-feature[enabled the Bulk Import feature and gave access to it].
+
+.Procedure
+. Click *Bulk Import* in the left sidebar.
+. Click the *Add* button in the top-right corner to see the list of all repositories accessible from the configured GitHub integrations.
+.. From the *Repositories* view, you can select any repository, or search for any accessible repositories.
+For each repository selected, a `catalog-info.yaml` is generated.
+.. From the *Organizations* view, you can select any organization by clicking on *Select* in the third column.
+This option allows you to select one or more repositories from the selected organization.
+. Click *Preview file* to view or edit the details of the pull request for each repository.
+The pull request details form is preloaded with some data, and you can also preview the content of the `catalog-info.yaml`.
+If the repository already has a `.github/CODEOWNERS` file, you can select the *Use CODEOWNERS file as Entity Owner* checkbox to use it, rather than having the `content-info.yaml` contain a specific entity owner.
+. Click *Save*.
```

Comment:
```suggestion
.. Edit the pull request description.
.. Review the content of the `catalog-info.yaml` file.
.. Optional: when the repository has a `.github/CODEOWNERS` file, you can select the *Use CODEOWNERS file as Entity Owner* checkbox to use it, rather than having the `content-info.yaml` contain a specific entity owner.
.. Click *Save*.
```